gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
```

Comment:
Can you also update `gradleVersion` in the root `build.gradle.kts`.
This is used by the `wrapper` task which is the [recommended way to upgrade gradle](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper).

Reply:
I already have :>

Reply:
oops, I'm blind :D